rest-api/src/main/resources/openapi.yml

```diff
@@ -0,0 +1,18 @@
+resourcePackages:
```

Comment:
Do we need to write our own openapi.yaml?

Reply:
This is for general information. But it was supposed to be .json... :thinking:

Reply:
and it was already added in b519b3da19e68006c08fcddc4a99113559f9df24. This file should not be here.